cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -73,8 +78,11 @@ def import_handler(request, course_key_string):
             # Do everything in a try-except block to make sure everything is properly cleaned up.
             try:
                 data_root = path(settings.GITHUB_REPO_ROOT)
-                course_subdir = "{0}-{1}-{2}".format(course_key.org, course_key.course, course_key.run)
-                course_dir = data_root / course_subdir
+                if library:
+                    courselike_subdir = "library-v1_{0}-{1}".format(course_key.org, course_key.run)
```

Comment:
Actually, it might be better to generate this from `unicode(course_key)`, with appropriate substitutions to make it valid on the filesystem. That way you get the `v1`, and you correctly escape the org and run.